test/functional/core/startup_spec.lua

```diff
@@ -40,6 +40,19 @@ describe('startup', function()
     ok(string.find(alter_slashes(meths.get_option_value('runtimepath', {})), funcs.stdpath('config'), 1, true) == nil)
   end)
 
+  it('prevents infinite loop', function()
+    clear({ '-u', 'NONE' })
```

Comment:
Default argv for `clear()` already contains `-u NONE`
```suggestion
    clear()
```